rllib/algorithms/utils.py

```diff
@@ -200,12 +193,13 @@ def _get_learner_bundles(config):
         else 0
     )
 
+    # aggregator actors are co-located with learners and use 1 CPU each
     bundles = [
         {
-            "CPU": config.num_learners
-            * (num_cpus_per_learner + config.num_aggregator_actors_per_learner),
-            "GPU": config.num_learners * config.num_gpus_per_learner,
+            "CPU": num_cpus_per_learner + config.num_aggregator_actors_per_learner,
+            "GPU": config.num_gpus_per_learner,
         }
+        for _ in range(config.num_learners)
```

Comment:
This looks more correct than before, but I am wondering, if this would still not ensure that `AggregatorActor`s being scheduled on the same node as we do not use placement groups. Could theoretically an EnvRunner be scheduled on CPUs of the same node instead of an `AggregatorActor`?